fabfile.py

```diff
@@ -21,7 +21,7 @@ def deploy(c, root, branch):
 
 @task(hosts=_HOSTS)
 def deploy_stable(c):
-    deploy(c, "/home/dolphin-emu/apps/www", "stable")
+    deploy(c, "/home/dolphin-emu/apps/www", "legacy")
```

Comment:
Please do not change this instance of "stable". It is referring to the `git` branch name on this repository, not Dolphin stable.

Reply:
any other instances like this?